frontend/src/helpers/connector.js

```diff
@@ -1,32 +0,0 @@
-import { InjectedConnector } from '@web3-react/injected-connector'
```

Comment:
why are you removing this file?

Reply:
You told me to remove connector in my PR